sal_interfaces/MTDome/MTDome_Commands.xml

```diff
@@ -59,6 +59,12 @@
   <SALCommand>
     <Subsystem>MTDome</Subsystem>
     <EFDB_Topic>MTDome_command_park</EFDB_Topic>
+    <Description>Move all components to park position and engage the brakes and locking pins.</Description>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>MTDome</Subsystem>
+    <EFDB_Topic>MTDome_command_goStationary</EFDB_Topic>
+    <Description>Stop all motion and engage the brakes. For an emergecy stop, use the stop command.</Description>
```

Comment:
Please explain *why* this command is not suitable for an emergency. My *guess* is that it stops azimuth motion more slowly than the `stop` command, and that all other actuators stop equally quickly for both commands. If that is the case then all those extra "This should also be used for an emergency stop" descriptions you added below are probably unnecessary.

Reply:
As discussed on Slack, I removed this command and added an `engage_brakes` parameter to all stop commands.